app/views/components/_banner.html.erb

```diff
@@ -12,18 +12,10 @@
   </p>
 <% end %>
 <section class="app-c-banner<% if aside %> app-c-banner--grid<% end %>" aria-label="Notice">
+  <%= content_block %>
   <% if aside %>
-    <div class="grid-row">
-      <div class="app-c-banner__main">
-        <%= content_block %>
-      </div>
-      <div class="column-third">
-        <div class="app-c-banner__aside">
-          <p><%= aside %></p>
-        </div>
-      </div>
+    <div class="app-c-banner__aside">
```

Comment:
I think the `aside` and the `desc` classes can probably be consolidated into one class applied to both paragraphs as they have similar styling.

Reply:
Sounds good to me

Reply:
@miaallers / @alextea can you make this change and push back onto this branch please?